docs/book/src/multiversion-tutorial/testdata/project/main.go

```diff
@@ -26,6 +26,7 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+
```

Comment:
Why space?

Reply:
External vs internal imports separation (internal = inside this project)

Reply:
The project has not this rule. 

> import (
>   std libs
> 
>   non-k8s libs
> 
>   k8s libs
> )

Reply:
generally we group all of k8s related imports together.